tests/Aspire.Azure.Messaging.EventHubs.Tests/AspireEventHubsExtensionsTests.cs

```diff
@@ -107,6 +108,209 @@ public void ProcessorClientShouldNotTryCreateContainerWithBlobContainerSpecified
+        }
+        else
+        {
+            s_clientAdders[clientIndex](builder, "eh", null);
+        }
+
+        using var host = builder.Build();
+
+        var assignedIdentifier = RetrieveClient(key, clientIndex, host) switch
+        {
+            EventProcessorClient processor => processor.Identifier,
+            EventHubConsumerClient consumer => consumer.Identifier,
+            EventHubProducerClient producer => producer.Identifier,
+            PartitionReceiver receiver => receiver.Identifier,
+            EventHubBufferedProducerClient producer => producer.Identifier,
+            _ => null
+        };
+
+        Assert.NotNull(assignedIdentifier);
+        Assert.Equal("customidentifier", assignedIdentifier);
+    }
+
+    [Theory]
+    [InlineData(false, EventHubProducerClientIndex)]
+    [InlineData(true, EventHubProducerClientIndex)]
+    [InlineData(false, EventHubConsumerClientIndex)]
+    [InlineData(true, EventHubConsumerClientIndex)]
+    [InlineData(false, EventProcessorClientIndex)]
+    [InlineData(true, EventProcessorClientIndex)]
+    [InlineData(false, PartitionReceiverIndex)]
+    [InlineData(true, PartitionReceiverIndex)]
+    [InlineData(false, EventBufferedProducerClientIndex)]
+    [InlineData(true, EventBufferedProducerClientIndex)]
+    public void BindsClientOptionsFromConfigurationWithConnectionStringAndEventHubName(bool useKeyed, int clientIndex)
+    {
+        var builder = Host.CreateEmptyApplicationBuilder(null);
+
+        ConfigureBlobServiceClient(useKeyed, builder.Services);
+
+        var key = useKeyed ? "eh" : null;
+        builder.Configuration.AddInMemoryCollection([
+            new KeyValuePair<string, string?>(
+                CreateConfigKey(
+                    $"Aspire:Azure:Messaging:EventHubs:{s_clientTypes[clientIndex].Name}",
+                    key, "ClientOptions:Identifier"), "customidentifier"),
+            new KeyValuePair<string, string?>(
+                CreateConfigKey(
+                    $"Aspire:Azure:Messaging:EventHubs:{s_clientTypes[clientIndex].Name}",
+                    key, "BlobContainerName"), "checkpoints"),
+            new KeyValuePair<string, string?>(
+                CreateConfigKey(
+                    $"Aspire:Azure:Messaging:EventHubs:{s_clientTypes[clientIndex].Name}",
+                    key, "BlobClientServiceKey"), useKeyed ? "blobs" : null),
+            new KeyValuePair<string, string?>(
+                CreateConfigKey(
+                    AspireEventHubsSection + s_clientTypes[clientIndex].Name, key, "PartitionId"), "foo"),            
+            new KeyValuePair<string, string?>(
+                CreateConfigKey(
+                    $"Aspire:Azure:Messaging:EventHubs:{s_clientTypes[clientIndex].Name}",
+                    key, "EventHubName"), "MyHub"),
+            new KeyValuePair<string, string?>("ConnectionStrings:eh", EhConnectionString),
+        ]);
+
+        output.WriteLine(builder.Configuration.GetDebugView());
```

Comment:
Do you want to keep these or did it just matter during dev.

Reply:
Personally, I prefer to keep it to make the test results more understandable for anyone else who wants to extend or clone them, but I guess it was only a dev thing.

Reply:
@radical Do you think this could be an issue in terms of space for logs or do we not care about it?

Reply:
@sebastienros Let me just kill the line if it's going to delay the merge. I'd like to make the 9.x cutoff.

Reply:
I've removed the lines and pushed. I'll resolve this now.